src/Rule.php

```diff
@@ -52,18 +52,16 @@ public function getParameters()
         return $this->params;
     }
 
-    public function setParameters(array $params)
+    public function setParameters($key, $value = null)
```

Comment:
but why?

Reply:
Create another method to `setParameter` or so.

Function should make only one thing - in this case:

- add one key-value to paramers
- add array of parameters